waspls/src/Wasp/Backend/ParseError.hs

```diff
@@ -0,0 +1,111 @@
+{-# LANGUAGE DeriveGeneric #-}
+
+module Wasp.Backend.ParseError
+  ( -- * Parse error
+    ParseError (..),
+
+    -- * Display functions
+    showError,
+    showErrorMessage,
+    errorSpan,
+
+    -- * Source positions
+    Span (..),
+    SourcePos (..),
+    offsetToSourcePos,
+  )
+where
+
+import Control.DeepSeq (NFData)
+import Data.List (intercalate)
+import GHC.Generics (Generic)
+import Wasp.Backend.Token (TokenKind)
+import qualified Wasp.Backend.Token as T
+import Wasp.Backend.TokenSet (TokenSet)
+import qualified Wasp.Backend.TokenSet as TokenSet
+
+data ParseError
+  = Unexpected !Span !TokenKind TokenSet
+  | UnexpectedEOF !Int TokenSet
+  deriving (Eq, Ord, Show, Generic)
+
+instance NFData ParseError
+
+data Span = Span !Int !Int deriving (Eq, Ord, Show, Generic)
```

Comment:
s/Span/Region ?

Reply:
Changed to Region to match wasp analyzer.